[2024-05-25T21:53:01.486+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T21:53:01.507+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:52:00+00:00 [queued]>
[2024-05-25T21:53:01.514+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:52:00+00:00 [queued]>
[2024-05-25T21:53:01.514+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-25T21:53:01.527+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first-step-el> on 2024-05-25 21:52:00+00:00
[2024-05-25T21:53:01.536+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1420) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T21:53:01.538+0000] {standard_task_runner.py:63} INFO - Started process 1422 to run task
[2024-05-25T21:53:01.539+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'first-step-el', 'scheduled__2024-05-25T21:52:00+00:00', '--job-id', '272', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmprhv39a8_']
[2024-05-25T21:53:01.541+0000] {standard_task_runner.py:91} INFO - Job 272: Subtask first-step-el
[2024-05-25T21:53:01.584+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:52:00+00:00 [running]> on host b2cfee8f0b2b
[2024-05-25T21:53:01.675+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='first-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T21:52:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T21:52:00+00:00'
[2024-05-25T21:53:01.677+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T21:53:01.685+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:53:01.697+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-25T21:53:01.699+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-25T21:53:01.704+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:53:01.715+0000] {logging_mixin.py:188} INFO - Inserts performed wihout fail
[2024-05-25T21:53:01.715+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T21:53:01.715+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T21:53:01.723+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=first-step-el, run_id=scheduled__2024-05-25T21:52:00+00:00, execution_date=20240525T215200, start_date=20240525T215301, end_date=20240525T215301
[2024-05-25T21:53:01.753+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T21:53:01.771+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-25T21:53:01.774+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
